src/shared/util/camera_calibration.cpp

```diff
@@ -775,6 +759,70 @@ CameraParameters::AdditionalCalibrationInformation::
   pointSeparation = new VarDouble("Points separation", 150);
 }
 
+void CameraParameters::AdditionalCalibrationInformation::updateControlPoints() {
+
+    std::vector<GVector::vector2d<double> > control_points = generateCameraControlPoints(
+            camera_index->getInt(), field->num_cameras_total->getInt(), field->field_length->getDouble(), field->field_width->getDouble());
+
+    if(control_points.empty())
+    {
+        return;
+    }
+
+    for (int i = 0; i < kNumControlPoints; ++i) {
+        control_point_field_xs[i]->setDouble(control_points[i].x);
+        control_point_field_ys[i]->setDouble(control_points[i].y);
+    }
+}
+
+std::vector<GVector::vector2d<double> > CameraParameters::AdditionalCalibrationInformation::
+generateCameraControlPoints(int cameraId, int numCamerasTotal, double fieldHeight, double fieldWidth) {
```

Comment:
This is a very helpful thing. It was never easy to get the control points right, even for experienced users.
Could you add images to the wiki which show the field and 1-8 cameras with their ID? I assume this helpful function will not work properly if one does not know which camera ID should be where.

Reply:
I added a picture to the wiki: https://github.com/RoboCup-SSL/ssl-vision/wiki/camera-calibration

I will also update the rest of the Wiki page later.

Reply:
Awesome, thanks!

Reply:
Documented the rest as well now: https://github.com/RoboCup-SSL/ssl-vision/wiki/camera-calibration